.papr.yml

```diff
@@ -16,7 +16,7 @@ timeout: 45m
 tests:
```

Comment:
You should be able to just drop the `packages:` block higher up since podman is already baked in FAH28.